mollie/api/objects/order.py

```diff
@@ -147,6 +146,18 @@ def create_refund(self, data, **params):
         refund = OrderRefunds(self.client).on(self).create(data, **params)
         return refund
 
+    def cancel_lines(self, data=None, **params):
+        """Cancel the lines given. When no lines are given, cancel all the lines.
+
+        We're using the create function,
+        because the API expects a POST request containing the lines you want to cancel.
```

Comment:
This is incorrect. The API expects a DELETE request with a body.

Reply:
Nee, in de comment zei hij dat hij een POST verwachte met de lines die verwijderd moeten worden: `With a POST request body containing the lines.`